static/includes/apps/AppInstallWizardACLConfiguration.md

```diff
@@ -1,20 +1,27 @@
 &NewLine;
 
-Select **Enable ACL** to show the ACL and ACE Entries options.
+Select **Enable ACL** to show the ACL and **ACE Entries** options.
 
-Enter or browse to select the dataset path. 
+{{< expand "Configuring ACE Entries" "v" >}}
+Enter or browse to select the dataset and populate **Host Path**.
+
+{{< trueimage src="/images/SCALE/Apps/InstallAppStorageConfigurationACLandACESettings.png" alt="Host Path ACL and ACE Settings" id="Host Path ACL and ACE Settings" >}}
+
+Next, click **Add** to the right of **ACL Entries** to show the permissions settings.
+Set **ID Type** to **Entry is for a USER** or **Entry is for a GROUP**.
+If you configured a group in TrueNAS that you want to give access to instead of a single user, set the ID to the group option and enter the GID for that group.
 
-Next click **Add** to the right of **ACL Entries** to show the permissions settings.
-Set **ID Type** to **Entry is for a USER**.
 Enter the UID as one of the following:
-* The default apps user which is either **473** for apps in the stable train or **568** for apps in the **enterprise** or **community** train.
-* The new user ID for the TrueNAS user created to serve as the app administrator.
+* The default apps user, which is **473** for apps in the stable train, **568** for apps in the **enterprise** or **community** trains, **999** for postgres storage volumes, or **0** if running as root.
+* The pre-assigned or run-as-user UID set as a default for the app.
+* The new user UID for a TrueNAS user created to serve as the app administrator.
```

Comment:
If user creates TrueNAS user with uid 3005, but the app runs as 568. This wont work.

The entry must match the run-as UID/GID.
(On apps that there is no fields in the UI to change, the UID can be seen on the Run As Context in the screen right before installing or right after installing)

Reply:
Changes made in the next commit.